src/bootstrap/README.md

```diff
@@ -111,6 +111,42 @@ compiler. What actually happens when you invoke rustbuild is:
 The goal of each stage is to (a) leverage Cargo as much as possible and failing
 that (b) leverage Rust as much as possible!
 
+## Incremental builds
+
+You can configure rustbuild to use incremental compilation. Because
+incremental is new and evolving rapidly, if you want to use it, you
+must replace the snapshot with a locally installed nightly build of
+rustc. You will want to keep this up to date.
+
+So, first thing is to install a nightly, presumably using
+`rustup`. You will then want to configure your directory to use this
+build, like so:
+
+```
+# you don't have to use a build directory, I just think it's nicer --nmatsakis
+> mkdir build
+> cd build
```

Comment:
Technically rustbuild already places all of it's output in a `build` directory, so this'll create `build/build`. Maybe we could leave the separate directory bits to a different section of the readme?